packages/passport/sdk/src/confirmation/confirmation.ts

```diff
@@ -180,8 +182,10 @@ export default class ConfirmationScreen {
         height: popupOptions?.height || CONFIRMATION_WINDOW_HEIGHT,
       });
       this.overlay = new Overlay(this.config.popupOverlayOptions);
-    } catch (e) {
+    } catch (error) {
       // If an error is thrown here then the popup is blocked
+      const errorMessage = error instanceof Error ? error.message : String(error);
+      trackError('passport', 'confirmationPopupDenied', new Error(errorMessage));
```

Comment:
One non-blocking suggestion, but we typically refer to the popup as being "blocked" rather than denied. It would be good to keep the naming consistent here.
```suggestion
      trackError('passport', 'confirmationPopupBlocked', new Error(errorMessage));
```